chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/PendingSurveillanceManager.java

```diff
@@ -92,19 +96,21 @@
     private SurveillanceUpdateValidator survUpdateValidator;
     private UserPermissionDAO userPermissionDAO;
     private CertifiedProductDAO cpDAO;
+    private Integer surveillanceThresholdToProcessAsJob;
 
     @Autowired
+    @SuppressWarnings("checkstyle:parameternumber")
     public PendingSurveillanceManager(Environment env, FileUtils fileUtils,
-            SurveillanceUploadManager survUploadManager, JobManager jobManager,
+            SurveillanceUploadManager survUploadManager, SchedulerManager schedulerManager,
             UserManager userManager, CertifiedProductManager cpManager, SurveillanceDAO survDao, UserDAO userDao,
             ActivityManager activityManager, CertifiedProductDetailsManager cpDetailsManager,
             SurveillanceCreationValidator survCreationValidator,
             @Qualifier("surveillanceUpdateValidator") SurveillanceUpdateValidator survUpdateValidator,
-            UserPermissionDAO userPermissionDAO, CertifiedProductDAO cpDAO) {
-        this.env = env;
+            UserPermissionDAO userPermissionDAO, CertifiedProductDAO cpDAO,
+            @Value("${surveillanceThresholdToProcessAsJob}") Integer surveillanceThresholdToProcessAsJob) {
```

Comment:
Are we still processing at a threshold, or are all of them going to a background job?

Reply:
I do see the threshold processing, so no issues here